claat/nodes/iframe_test.go

```diff
@@ -0,0 +1,71 @@
+package nodes
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestNewIframeNode(t *testing.T) {
+	tests := []struct {
+		name string
+		in   string
```

Comment:
Consider renaming `in` to `url`.

Reply:
I find this makes it clearer at a glance what the inputs and outputs are. What about `inURL`?

Reply:
I would prefer `inURL` over `in` and `url` over either. I trust your judgement.

Reply:
inURL works as a compromise I think. I'll propagate that format over the three open PRs as well.